google/cloud/aiplatform/explain/lit.py

```diff
@@ -244,12 +340,17 @@ def create_lit_model(
     input_types: "OrderedDict[str, lit_types.LitType]",  # noqa: F821
     output_types: "OrderedDict[str, lit_types.LitType]",  # noqa: F821
     attribution_method: str = "sampled_shapley",
+    model_id: Optional[str] = None,
```

Comment:
This function may be getting a overloaded with `model` argument representing both model and endpoints. Recommend breaking out the endpoint support to a new function: `create_lit_model_from_endpoint`.

Reply:
Done.